app/controllers/questions/questions_controller.rb

```diff
@@ -98,6 +98,10 @@ def redirect_if_completed_intake_present
       end
     end
 
+    def set_in_intake_flow
+      @in_intake_flow = true
```

Comment:
what does this do?

Reply:
also, you could maybe define this as a helper method instead 

```
helper_methods :in_intake_flow

def in_intake_flow
  true
end
```

then, if this method is trying to get used somewhere and its not defined it will raise an error